osmaxx/excerptexport/static/excerptexport/scripts/maps/draw_controls.js

```diff
@@ -49,7 +49,7 @@ var draw_controls = function (map) {
     }
 
     function showSizeOfLayer(layer) {
-        var allowedMaxSize = 5 * 1000 * 1000 * 1000;  // 5 GB in bytes, approx. Germany
+        var allowedMaxSize = 1000 * 1000 * 1000;  // 1 GB in bytes, approx. 1/5 of Germany
```

Comment:
Should we use base 1000 or 1024?

In the comment, maybe write that this comparison is for PBFs.